modules/llama-stack-apis.adoc

```diff
@@ -0,0 +1,91 @@
+:_module-type: REFERENCE
+[id="llama-stack-apis_{context}"]
+= Llama Stack APIs 
+
+The listed APIs are included in Llama Stack that are not OpenAI-compatible. 
+
+== Supported Llama Stack APIs in {productname-short}
+
+=== Agents API
+* *Endpoint:* `/v1alpha/agents`.  
+* *Providers:* All agent backends deployed through OpenShift AI.
+* *Support level:* Developer Preview
+
+The Agents API allows you to create and manage AI agents. 
+
+---
+
+=== Datasets_IO API 
+* *Endpoint:* `/v1beta/datasetio`.  
+* *Providers:* All dataset_io backends deployed through OpenShift AI.
+* *Support level:* Technology Preview
+
+The Dataset_IO API manages the input and output of datasets and its content.
+
+---
+
+=== Datasets API
+* *Endpoint:* `/v1beta/datasets`
+* * *Providers:* All dataset backends deployed through OpenShift AI.
+* *Support level:* Technology Preview
+
+The Datasets API manages creating, deleting, and listing datasets themselves.
+
+---
+
+=== Evaluation API
+* *Endpoint:* `/v1beta/eval`
+* *Providers:* All evaluation backends deployed through OpenShift AI.
+* *Support level:* Developer Preview
+
+The Evaluation API defines an evaluation task for models and datasets
+
+---
+
+=== Inference API
+* *Endpoint:* `/v1alpha/inference`
+* *Providers:* All inference backends deployed through OpenShift AI.
+* *Support level:* Developer Preview
+
+[WARNING]
+====
+The majority of the Inference API is deprecated. The Inference providers use the Completions and Chat Completions APIs now.
+====
+
+The Inference API enables conversational, message-based interactions with models served by Llama Stack in OpenShift AI.
+
+---
+
+=== Safety API 
+* *Endpoint:* `/v1/safety`
+* *Providers:* All safety backends deployed through OpenShift AI.
+* *Support level:* Technology Preview
+
+The Safety API detects and prevent harmful content in model inputs and outputs.
+
+---
+
+=== Tool Runtime API
+* *Endpoint:* `/v1/tool-runtime`
+* *Providers:* All safety backends deployed through OpenShift AI
+* *Support level:* Developer Preview 
+
+The Tool Runtime API allows a model to dynamically to call a tool at runtime.
+
+---
+
+=== Vector_IO API
+* *Endpoint:* `/v1/vector-io`
+* *Providers:* All Vector_IO backends deployed through OpenShift AI
+* *Support level:* Developer Preview 
+
+The Vector_IO API allows you to manage and query vector embeddings: numeric representations of data. 
+
+---
+
+=== Models API 
```

Comment:
I don't know If we should include these admin apis. or if we are: we also need `/v1/providers` and `/v1/inspect`

Reply:
I kind of agree, I think it could confusion then be helpful as documentation. @nathan-weinberg do you think so as well? 
So the only ones this list should really have is:
- Agents API 
- Dataset_IO APU 
- Evaluations API 
- Inference API
- Safety API
- Tool_runtime API 
- Vector_IO r